README.md

```diff
@@ -1,25 +1,25 @@
 # Werewolf-Brain
 **Werewolf-Brain** (or **wwb**) is a JavaScript library 
-for creating scenarios for playing the party card game 
+for the creation of *scenarios* to play the party card game 
 [Ultimate Werewolf](https://en.wikipedia.org/wiki/Ultimate_Werewolf "Wikipedia").
-There are several ways to craft funny games with this 
-library, check out the examples section.
+There are several ways to craft fun games with this 
+library. For more on usage, check out the examples section below.
 
 ## Installing
 If you use npm, ```npm install werewolf-brain```. Otherwise, 
 download the files from [here](https://github.com/wistcc/werewolf-brain "Github") and add them to the project.
 
 ## What does it do?
-Creates new scenarios based on the number of players, on a 
-template, or the available cards. The number of players is not 
-limited. If the current deck of cards can't have a game with all
-the players, the library will give you a game with the maximum 
-number of players that could found in 5000 tries. 
-
-We use a game object for describing a scenario. This object contains 
-the number of players, the game weight and the deck itself. The game 
+Creates new scenarios based on the number of players,  or available cards, given a template. 
+Moderators can then use these scenarios to set up Werewolf games. 
+The number of players is not limited. If the current deck of cards 
+can't have a game with all the players, the library will give you a
+game with the maximum number of players that it could calculate in 5000 tries. 
+
+We use a **game** object to describe a scenario. This object contains 
+the number of *players*, the game *weight* and the *deck* itself. The game 
 weight gives you an idea of how balanced is the game. Positive weight 
-means a game in favor to the villagers, negatives means the opposite.
+means a game in favor to the Villagers, negatives means the opposite: a game in favor to the Werewolves.
```

Comment:
Then I suggest either a change in the _opposite_ adjective, or in a change in the preceding proposition.

In the original form, what would exactly be 'the opposite'? i. e. what would be the opposite of a game in favor to the villages?